### DISCUSSION TITLE
Does Nightdriver support (or plan to support) APA102/SK9822 LED Strips

### DISCUSSION THREAD

**Discussion by vbisbest** [PlummersSoftwareLLC/NightDriverStrip#589] [Q&A]
Love the project, just hoping it works on APA102 strips.  Thanks!

**Comment by rbergen** [comment]
I'm converting this to a Q&A discussion.

**Comment by rbergen** [comment]
@vbisbest Short answer: no, but it could. Personally I've only tested anything with WS2812B LEDs (matrices in my case) and a HUB75 panel. 

The actual controlling of the WS2812B LEDs is done through the [FastLED](https://fastled.io/) library, which apparently also supports APA102 strips. However, as it stands the use of WS2812 is hardcoded (for instance, in [ledstripgfx.h](https://github.com/PlummersSoftwareLLC/NightDriverStrip/blob/cbc933c4df9d1744728a0c02fc2ae2251f4a9981/include/ledstripgfx.h#L49)).

So, I guess someone would have to make the LED protocol configurable (or `#define`-able) within the code and then confirm that FastLED does what is expected when it's told the connected strip(s) it's driving is/are of the APA102 type.
